packages/server/src/middleware/nuxt.js

```diff
@@ -38,7 +38,10 @@ export default ({ options, nuxt, renderRoute, resources }) => async function nux
 
     // Add ETag header
     if (!error && options.render.etag) {
-      const etag = generateETag(html, options.render.etag)
+      const etagArg = { value: undefined }
+      await nuxt.callHook('render:generateETag', html, etagArg)
```

Comment:
Should we pass the context as well here?

I didnt add it as users either already define the hook in nuxt.config and module imlementation can already access nuxt options from the module container